cernopendata_client/cli.py

```diff
@@ -41,6 +41,7 @@
 
 @click.group()
 def cernopendata_client():
+    """Implement the group."""
```

Comment:
Note that this does not look right in the user help `cernopendata-client --help`.  It needs to be a concise message about what the client does.

Reply:
okay, Adding more info 👍

Reply:
Can you please suggest what should be appropriate here. Thanks :)